packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/SelectedItemWithMenu.tsx

```diff
@@ -33,7 +33,7 @@ export class SelectedItemWithMenu extends React.Component<IPeoplePickerItemWithM
         <div className='ms-PickerItem-content'>
           <Persona
             { ...item }
-            presence={ item.presence ? item.presence : PersonaPresence.online }
+            presence={ item.presence != null ? item.presence : PersonaPresence.none }
```

Comment:
`!== undefined` means they didn't pass a value in. 

The enum will result in a numerical value and if, say, PersonaPresence.online were value 0, this would break. Right? It would be best to say "if they provide it, use that value. Otherwise fallback to the null value."